packages/@okta/i18n/src/properties/login.properties

```diff
@@ -685,8 +685,11 @@ deviceTrust.universalLink.fallback.getOktaMobile.subtitle = Go to the {0}AppStor
 loopback.polling.cancel.link = Cancel and take me to sign in
 customUri.subtitle = Launching Okta Verify...
 customUri.content = <div class="skinny-content"> If nothing prompts from the browser, <a href="#" id="launch-ov" class="link">click here</a> to launch Okta Verify, or make sure Okta Verify is installed.</div>
-customUri.required.content.p1 = Please click&nbsp;<span class="strong">Open oktaverify.app</span>&nbsp;if you see the system dialog.
-customUri.required.content.p2 = If nothing prompts from the browser,&nbsp;<a href="#" id="launch-ov" class="link">click here</a>&nbsp;to launch Okta Verify, or&nbsp;<a href="{0}" target="_blank" id="download-ov" class="link">download & run Okta Verify</a>.
+customUri.title = Click "Open Okta Verify" on the browser prompt
+customUri.required.content.p1 = Didn’t get a prompt?
+customUri.required.content.p2 = Don’t have Okta Verify?
+customUri.required.content.p3 = <a href="{0}" target="_blank" id="download-ov" class="link">Download here</a>
```

Comment:
The only text that needs to be translated here is `Download here`. Let's move the anchor tag out into the template.